spikes/di.ts

```diff
@@ -1 +1,68 @@
 // Copyright 2024-2024 the API framework authors. All rights reserved. MIT license.
+
+import "npm:reflect-metadata";
```

Comment:
You might have said this before. Is the vision here to support an inversify-like API? As Inversity depends on experimental decorators :)